gruntfile.js

```diff
@@ -13,6 +13,23 @@ module.exports = function (grunt) {
 
     require('time-grunt')(grunt);
 
+    var concatConfig = {
+        target: {
+            src: [
+                'src/**/*.js'
+            ],
+            dest: 'dist/angular-token-auth.js'
+        }
+    };
+
+    var uglifyConfig = {
+        target: {
+            files: {
+                'dist/angular-token-auth-mini.js': 'dist/angular-token-auth-mini.js'
```

Comment:
The file naming convention is `foo-bar.js` and `foo-bar.min.js`, so here it would be `angular-token-auth.js` and `angular-token-auth.min.js`